### PR TITLE
chore: PX-127 Add Portuguese and Greek locale options to Hosted Payments Page and Payment Links

### PR DESCRIPTION
**Pull request by cko-castiel-ho**
# Description of changes in PR

- Added new `pt-PT` and `el` locale options to both ABC and NAS references of Hosted Payments Page and Payment Links

## Checklist

- [x] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [x] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

**Review by punit-shah-cko — DISMISSED** [review]

File: nas_spec/changelog.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # Changelog
 
-| Date       | Description of change                                                                                                 
+| Date       | Description of change                                                                                                 |
 |------------|-----------------------------------------------------------------------------------------------------------------------|
-| 2023/02/01 | Update file uploads/retrievals for Platforms.
+| 2023/02/07 | Added Portuguese and Greek locale options to Hosted Payments Page and Payment Links                                   |
+| 2023/02/01 | Update file uploads/retrievals for Platforms.                                                                         |
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/changelog.md:6]:
```suggestion
| 2023/02/01 | Update file uploads/retrievals for Platforms                                                                       |
```

Comment (cko-castiel-ho) [reply nas_spec/changelog.md:6]: any concern if I update a past change log entry?

Comment (eoin-shaughnessy-cko) [reply nas_spec/changelog.md:6]: No, I think it will be fine as it does not change the meaning of the content. :)

Comment (eoin-shaughnessy-cko) [reply nas_spec/changelog.md]: Thank you!

**Review by eoin-shaughnessy-cko — APPROVED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -3,5 +3,5 @@
 /abc_spec/components/schemas/Payments/ @checkout/gateway-product-cko
 /nas_spec/components/schemas/Payments/ @checkout/gateway-product-cko
 
-/nas_spec/components/schemas/PaymentLinks/ @checkout/payment-interfaces-cko
-/nas_spec/components/schemas/HostedPayments/ @checkout/payment-interfaces-cko
\ No newline at end of file
+/nas_spec/components/schemas/PaymentLinks/ @checkout/payment-experience-cko
```
Comment (cko-castiel-ho) [inline .github/CODEOWNERS:6]: ownership of Hosted Payments Page and Payment Links has been transferred to the new Payment Experience team, updating code owners here

**Review by punit-shah-cko — APPROVED** [review]

**Review by eoin-shaughnessy-cko — APPROVED** [review]

**Pull request merged**